.github/workflows/release.yml

```diff
@@ -38,20 +37,11 @@ jobs:
           sudo dpkg -i goreleaser_amd64.deb
           rm goreleaser_amd64.deb
 
-      - name: Set GoReleaser flags
-        id: set-goreleaser-flags
-        run: |
-          if ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags') }} ; then
-            echo "::set-output name=GORELEASER_FLAGS::--rm-dist --skip-validate"
-          else
-            echo "::set-output name=GORELEASER_FLAGS::--rm-dist --snapshot --skip-publish --skip-validate"
-          fi
-
       - name: Build packages
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          goreleaser release ${{ steps.set-goreleaser-flags.outputs.GORELEASER_FLAGS }}
+          goreleaser release --rm-dist --skip-validate
```

Comment:
We have two cases aside of workflow_dispatch: pushing the `master` branch and pushing a tag. The latter was not changed.

However pushing to the `master` branch now doing something strage. It builds packages:

- Without `--snapshot`: I guess it'll look incorrectly in the artifacts.
- Without `--skip-publish`: I guess it'll try to update the releases page.

Reply:
Sorry, I thought flags only differ for workflow dispatch. Restored old behaviour